### PR TITLE
Reg user files finished

### PR DESCRIPTION
**Pull request by atvick21**
Refactoring of registered user files should be good now

### CODE REVIEW

File: src/main/java/edu/metrostate/ics499/team2/services/RegisteredUserService.java [hunk 1]
```diff
@@ -38,6 +38,12 @@ public RegisteredUser getByUserEmailAndPassword(String email, String password) {
 	}
 
 	public RegisteredUser create(@RequestBody final RegisteredUser registeredUser) {
-		return userRepo.save(registeredUser);
+		return isValid(registeredUser) ? userRepo.save(registeredUser) : null;
+	}
+	
+	@Override
+	public boolean isValid(RegisteredUser obj) {
```
Comment (dnguyen000) [inline src/main/java/edu/metrostate/ics499/team2/services/RegisteredUserService.java:45]: Did you happen to test this method? Wanted to make sure the logic to ensure uniqueness works in your use-case.

**Comment by calebTree** [comment]
I'm confused how these 7 commits: 38ac063, cb18b93, 5a2237d, 6d5ac17, 8795f32, 3e52cac, e699dae are already merged into main?

**Comment by calebTree** [comment]
The two new commits: ee17f4a, 05613a4 look good.

**Review by calebTree — APPROVED** [review]
The two new commits: ee17f4a, 05613a4 look good.

**Review by f947 — APPROVED** [review]

**Review by dnguyen000 — APPROVED** [review]

**Review by wale0 — APPROVED** [review]

**Pull request merged**